code/mobile/android/PhoneVR/keystore.properties

```diff
@@ -0,0 +1,4 @@
+storePassword=RohitKey@Store123
```

Comment:
Please don't use Personal names/addresses/etc etc info. And keeping passwords in open text is not idea, you may use Appveyor Secure system to inject these vars at build time ([example](https://stackoverflow.com/questions/29679654/access-gradle-command-line-p-variables-in-android-build)).